tools/sz_repo_cli/lib/src/commands/src/pub_get_command.dart

```diff
@@ -55,9 +55,7 @@ class PubGetCommand extends Command {
 
     final res = taskRunner
         .runTaskForPackages(
-          packageStream: repo
-              .streamPackages()
-              .where((package) => package.hasTestDirectory),
```

Comment:
lol

Reply:
I copied it from the test_command code I think and forgot to remove this 🤡  
I mean should've been catched by the reviewer of the PR where I added this obviously ;P